### DISCUSSION TITLE
May 18th 2023 Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#19548] [Community Call]
Agenda:
- PowerShellGet 7.4 update @SydneyhSmith 
- PowerShellGet beta release @anamnavi 
- Microsoft Build - @SydneyhSmith 
- PSReadLine Beta - @StevenBucher98 
- Completion Predictor new release @StevenBucher98
- VSCode Extension Updates @andschwa 
- Community Day Updates @SteveL-MSFT 
- Docs Update @sdwheeler 
- Conference Recap @StevenBucher98 @SydneyhSmith @maertendMSFT @theJasonHelmick @sdwheeler

**Comment by insidethedarkcity** [comment]
Hello @StevenBucher98,

I think I can demo a bit of a cross-tech thing, which includes .net interactive kernel, jupyter notebooks in vs code, some azure and IP management suff on top of those.

**Comment by StartAutomating** [comment]
Assuming I join the call, I could demonstrate EZOut, PipeScript, or ugit.

**Comment by JustinGrote** [comment]
I can demo my websocket custom transport at the end if time allows.

**Reply by SteveL-MSFT** [reply]
@JustinGrote we can have you as first demo for next month if that works for you

**Reply by JustinGrote** [reply]
> @JustinGrote we can have you as first demo for next month if that works for you

I'll be somewhere in Germany for that one prior to psconfeu so let's bump it another month :). Sorry I missed the majority of the call, last minute work thing :/

**Comment by alexandair** [comment]
If SSH is the future of PowerShell remoting, shouldn't it be enabled and configured by default? (In the same way how WinRM-based remoting was enabled in Windows Server 2012.)

**Reply by AspenForester** [reply]
Didn't Danny say that was on the roadmap for Windows Server?

**Reply by alexandair** [reply]
I might have missed that.

**Reply by SteveL-MSFT** [reply]
Our goal has been to make SSH the default remoting protocol in Windows and Azure.  Anything in Windows will take longer than you'd expect because the requirements are much higher.

**Comment by tikicoder** [comment]
I will have to try to join next month
Here is something I have been working on.

Here is my base windows config
https://github.com/tikicoder/samples/tree/main/computer_config/windows

Has some other scripts up there but figured I would share that.

**Reply by StevenBucher98** [reply]
Add this to next months discussion! https://github.com/orgs/PowerShell/discussions/19675
